apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -88,6 +88,11 @@ type ClusterQueueSpec struct {
 	// preempt to accomomdate the pending Workload, preempting Workloads with
 	// lower priority first.
 	Preemption *ClusterQueuePreemption `json:"preemption,omitempty"`
+
+	// resourceMetrics enables the reporting of resource usage and quotas
+	//
+	// +optional
+	ResourceMetrics *bool `json:"resourceMetrics,omitempty"`
```

Comment:
This configuration should be global and live in the `Configuration` object.

Reply:
In my opinion this provides better control of the metrics.

Reply:
let's start simple, just one global check.

Reply:
done